packages/router-plugin/tests/router.plugin.spec.ts

```diff
@@ -23,70 +12,19 @@ import {
   ofActionSuccessful,
   State,
   Action,
-  StateContext,
-  Select,
-  Selector
+  StateContext
 } from '@ngxs/store';
 
 import {
   NgxsRouterPluginModule,
   RouterState,
   RouterStateSerializer,
   Navigate,
-  RouterNavigation,
-  RouterStateModel,
-  RouterDataResolved
+  RouterNavigation
 } from '../';
 
 describe('NgxsRouterPlugin', () => {
-  it('should dispatch router state events', fakeAsync(async () => {
-    // Arrange
-    createTestModule();
-    const router: Router = TestBed.get(Router);
-    const store = TestBed.get(Store);
-    const log = logOfRouterAndStore(router, store);
-
-    // Act
-    await router.navigateByUrl('/');
-
-    // Assert
-    expect(log).toEqual([
-      { type: 'url', state: undefined }, // init event. has nothing to do with the router
-      { type: 'router', event: 'NavigationStart', url: '/' },
-      { type: 'router', event: 'RoutesRecognized', url: '/' },
-      { type: 'router', event: 'GuardsCheckStart', url: '/' },
-      { type: 'url', state: '/' }, // RouterNavigation event in the store
-      { type: 'router', event: 'GuardsCheckEnd', url: '/' },
-      { type: 'router', event: 'ResolveStart', url: '/' },
-      { type: 'router', event: 'ResolveEnd', url: '/' },
-      { type: 'router', event: 'NavigationEnd', url: '/' }
-    ]);
-  }));
-
-  it('should dispatch router state events for another route', fakeAsync(async () => {
```

Comment:
Shouldn't we restore this test?

Reply:
I'm not sure. This test is not explicit and we always have to fix it when we do any change. I don't think it's a good approach.

Reply:
I think we could restore this test but run expectations on other things, not logs.

Reply:
See other comment about Characterization tests